### PR TITLE
bump cosmos sdk

### PR DESCRIPTION
**Pull request by faddat**
This PR bumps cosmos-sdk to v0.45.6

🎏 : this is odd

### CODE REVIEW

**Comment by faddat** [comment]
oh wait it is just this:

https://github.com/cosmos/gaia/issues/1449

**Comment by faddat** [comment]
* the hub community elected me.  I won't stop making PR's.
* #1449 is the cause of this issue.  Specifically the silly diff thing that doesn't help anything and does break ci.  I will fix v7

**Comment by faddat** [comment]
I may be wrong and this could be unrelated to #1449 and that is bad.  That would indicate that v0.45.6 isn't in. fact a drop-in replacement.

File: .github/workflows/sims.yml [hunk 1]
```diff
@@ -105,22 +55,10 @@ jobs:
...
-        with:
-          PATTERNS: |
-            **/**.go
-            go.mod
-            go.sum
-      - uses: actions/cache@v2.1.7
-        with:
-          path: ~/go/bin
-          key: ${{ runner.os }}-go-runsim-binary
-        if: "env.GIT_DIFF != ''"
```
Comment (tac0turtle) [inline .github/workflows/sims.yml]: lets keep the scope of this pr small. Can we spin these changes out into another PR?

at least the removal of things, the bump in go version is needed

**Review by tac0turtle — APPROVED** [review]
LGTM, left one comment on scope

Why is dependabot not working here? @okwme do you know

lets get this in, gaia is two releases behind

**Comment by tac0turtle** [comment]
so this is actually blocked on the liquidity module, they need to update first. @okwme does the gaia team have to fork that to maintain it till its fully deprecated?

**Comment by faddat** [comment]
Thanks a ton Marko!

**Comment by faddat** [comment]
https://github.com/Gravity-Devs/liquidity/pull/18

**Comment by okwme** [comment]
thanks @faddat , looks like @dongsam made a release of liquidity with v0.45.6. I can't make a suggestion on that line of the go.mod file on your PR, could you update with the new release of liquidity?
also please remove the files not relevant to a version bump.

**Comment by faddat** [comment]
sure

**Comment by glnro** [comment]
I've had this running on a node that was previously running `7.0.1` for over 48 hours with no issues. Has anyone tested state syncing from this version as well?

**Comment by okwme** [comment]
thanks @faddat, i'm confused by the CI errors.
When I clone and build locally i don't get any nor when i run golangci-lint locally.
could these have to do with the CI changes I merged earlier today from you?

**Comment by tac0turtle** [comment]
~~this flow is a bit backwards, it should land on main then be back ported right?~~

nvm you guys are working on rcs

**Comment by faddat** [comment]
@okwme I, too am confused by the ci errors.

**Comment by okwme** [comment]
closing in favor of this PR: https://github.com/cosmos/gaia/pull/1584
I went through and updated everything that wasn't go v1.18 compatible in the CI and it's passing now
